### PR TITLE
feat: [EXT-3599] Expose plain client type

### PR DESCRIPTION
**Pull request by colomolo**
## Summary
Properly exposing `PlainClientAPI` type.

### CODE REVIEW

**Comment by marcolink** [comment]
@colomolo out of curiosity, will this also have an effect on the generated docs?

**Comment by colomolo** [comment]
> @colomolo out of curiosity, will this also have an effect on the generated docs?

I don't know, honestly. Which docs do you mean?

**Comment by marcolink** [comment]
> > @colomolo out of curiosity, will this also have an effect on the generated docs?
> 
> I don't know, honestly. Which docs do you mean?

https://contentful.github.io/contentful-management.js/contentful-management/10.37.0/

**Comment by colomolo** [comment]
> > > @colomolo out of curiosity, will this also have an effect on the generated docs?
> > 
> > 
> > I don't know, honestly. Which docs do you mean?
> 
> https://contentful.github.io/contentful-management.js/contentful-management/10.37.0/
> 
> ... you can test it by running [this](https://github.com/contentful/contentful-management.js/blob/master/package.json#L27) script locally

Yes, it's there.

File: lib/export-types.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export * from './common-types'
+export * from './plain/common-types'
```
Comment (andipaetzold) [inline lib/export-types.ts:2]: export all or just that specific type?  Atm it's the same, but in the future we might export more types from `common-types`

**Review by andipaetzold — APPROVED** [review]

**Review by marcolink — APPROVED** [review]

File: lib/export-types.ts (hunk 1 above)
Comment (colomolo) [reply lib/export-types.ts:2]: All of them. Hence the name 'common-types'

**Comment by andipaetzold** [comment]
Before we merge: We are currently in the weird situation where we export types from `types` and from the root. `PlainClientAPI` is already exported from the root. Probably something we should eventually fix by exporting everything from the root.

-> It's already possible to import the type without this PR.

**Comment by colomolo** [comment]
> Before we merge: We are currently in the weird situation where we export types from `types` and from the root. `PlainClientAPI` is already exported from the root. Probably something we should eventually fix by exporting everything from the root.
> 
> -> It's already possible to import the type without this PR.

Yeah, this is not only about `PlainClientAPI`, but relates to all exported types. So I think we should consider a broader solution for this root vs '/types' export thing.

**Pull request merged**